tools/modules/framework/interface_package.sh

```
@@ -0,0 +1,171 @@
+	["see_current_apt,desc"]="Check when apt list was last updated and suggest updating or update"
+	["see_current_apt,example"]="update"
+	["see_current_apt,doc_link"]=""
+	["see_current_apt,group"]="Interface"
+)
+#
+# Function to check when the package list was last updated
+#
+see_current_apt() {
+	# Number of seconds in a day
+	local update_apt="$1"
+	local day=86400
```

Shellcheck (0.10.0)

[warning] day appears unused. Verify use (or export if used externally).

(SC2034)